test/src/specs/sync/chain_forks.rs

```diff
@@ -10,9 +10,9 @@ use ckb_types::{
     prelude::*,
     H256,
 };
-use failure::_core::time::Duration;
```

Comment:
[According to the `failure`](https://github.com/rust-lang-nursery/failure/blob/135e2a3b9af422d9a9dc37ce7c69354c9b36e94b/src/lib.rs#L31-L34):

```rust
// Re-export libcore using an alias so that the macros can work without
// requiring `extern crate core` downstream.
#[doc(hidden)]
pub extern crate core as _core;
```